code/modules/tgui/external.dm

```diff
@@ -55,10 +55,10 @@
  * * ui_key - Key of the UI to be updated. (OPTIONAL)
  */
 /datum/proc/update_static_data(mob/user, datum/tgui/ui, ui_key = "main")
-	ui = SStgui.try_update_ui(user, src, ui_key, ui)
-	// If there was no ui to update, there's no static data to update either.
 	if(!ui)
-		return
+		ui = SStgui.try_update_ui(user, src, ui_key, ui)
+		if(!ui)
+			return // If there was no ui to update, there's no static data to update either.
 	ui.push_data(null, ui_static_data(user), TRUE)
```

Comment:
This is unrelated and shouldn't actually be in this PR, it's a leftover of some other experimentation I was doing. It *is* a code improvement though so I might just leave it.